### PR TITLE
task: SCBE-671 Adds repeat_every to drawings

### PR DESCRIPTION
**Pull request by JoelShinness-Unity**
* Adds `repeat_every` to drawings throughout the documentation

Screen Shot 2021-04-29 at 3 47 28 PM
Screen Shot 2021-04-29 at 3 47 32 PM
Screen Shot 2021-04-29 at 3 48 02 PM
Screen Shot 2021-04-29 at 3 48 14 PM
Screen Shot 2021-04-29 at 3 48 22 PM
Screen Shot 2021-04-29 at 3 48 27 PM
Screen Shot 2021-04-29 at 3 48 31 PM
Screen Shot 2021-04-29 at 3 48 35 PM

### CODE REVIEW

**Comment by dkaga** [comment]
@JoelShinness-FFATG Do you describe what unit this number is measured in?

File: source/index.html.md [hunk 1]
```diff
@@ -2101,7 +2104,8 @@ This endpoint is use to returns a list of subscription entries.
       "tickets_count": 1000,
       "ticket_price": "25.00",
       "currency": "NOK",
-      "cover_url": null
+      "cover_url": null,
+      "repeat_every": 604800
```
Comment (justinhamadeunity) [inline source/index.html.md:2108]: @JoelShinness-FFATG Can we add somewhere to the doc what the metric is?  I assume it is seconds?

**Review by justinhamadeunity — CHANGES_REQUESTED** [review]
Is repeats_every in seconds?

File: source/index.html.md (hunk 1 above)
Comment (JoelShinness-Unity) [reply source/index.html.md:2108]: It is seconds.  I didn't see a place to add the note other than "number", but I'll take another look.

**Review by justinhamadeunity — APPROVED** [review]
Approving with a note that we need to add more descriptions to the documentation later

**Pull request merged**